server/parsec/api.py

```diff
@@ -74,7 +81,7 @@ def api(fn: ApiFnWithSelf) -> ApiFnWithSelf:  # pyright: ignore[reportMissingTyp
         case _:
             raise ValueError(f"unexpected family {m_family}")
     assert types["client_ctx"] == ForwardRef(expected_type_name), (
-        f"Expect `client_ctx` to be an `{expected_type_name}`"
+        f"Expect `client_ctx`({types['client_ctx']}) to be an `{expected_type_name}`. Did you add `from __future__ import annotations` ?"
```

Comment:
I don't understand the ``` Did you add `from __future__ import annotations` ? ``` part, can you explain ?

Reply:
before I added that to the account component it was failing because client ctx was a forward ref, not the actual client context. I spend more time that I wanted to on that, and I'm pretty sure I already encountered this error while setting up the shamir component. Maybe I can just put a comment describing that  before the assert and not include it in the error message.

Reply:
The original discussion: https://github.com/Scille/parsec-cloud/pull/10174#discussion_r2044984114

Reply:
> Maybe I can just put a comment describing that before the assert and not include it in the error message.

Yes, just adding as comment with the error message from https://github.com/Scille/parsec-cloud/pull/10174#discussion_r2044984114 is a good idea